zebedee-cms/src/main/java/com/github/onsdigital/zebedee/service/ZebedeeImageService.java

```diff
@@ -0,0 +1,41 @@
+package com.github.onsdigital.zebedee.service;
+
+import com.github.onsdigital.dp.image.api.client.ImageClient;
+import com.github.onsdigital.dp.image.api.client.exception.ImageAPIException;
+import com.github.onsdigital.dp.image.api.client.model.Image;
+import com.github.onsdigital.dp.image.api.client.model.Images;
+import com.github.onsdigital.zebedee.model.Collection;
+
+import java.io.IOException;
+
+/**
+ * Image related services
+ */
+public class ZebedeeImageService implements ImageService {
```

Comment:
No need to call it `ZebedeeImageService` - we're in Zebedee so that is a given. `ImageServiceImpl` would follow the naming convention for most interface implementations in this codebase.

Reply:
I was keeping it consistent with the existing `ZebedeeDatasetService` which it is based on but I agree it is a better name. I'm renaming it to your suggestion.